.github/workflows/test.yml

```diff
@@ -15,7 +18,25 @@ jobs:
       image: ghcr.io/bdaiinstitute/spot_ros2_jammy_humble:main
     steps:
     - uses: actions/checkout@v3
-    - name: Build and run tests
+    - name: Build package
       run: |
-        source /opt/ros/humble/setup.bash
+        source /opt/ros/$ROS_DISTRO/setup.bash
+        pwd 
```

Comment:
Consider cleaning up a little bit here. Do you need the pwd? I think GHA has wget and pip by default?

Reply:
I removed pwd and pip. But I kept apt update and the wget, it was errorring otherwise if I removed any of these two lines.
`wget: command not found`